README.md

```diff
@@ -9,73 +10,96 @@ you have permission to install apps. **Please note that the features in this
 project require that the workspace be part of
 [a Slack paid plan](https://slack.com/pricing).**
 
-### Install the Slack CLI
+### Developer Program
 
-To use this template, you need to install and configure the Slack CLI.
-Step-by-step instructions can be found in our
-[Quickstart Guide](https://api.slack.com/automation/quickstart).
+Join the [Slack Developer Program](https://api.slack.com/developer-program) for
+exclusive access to sandbox environments for building and testing your apps,
+tooling, and resources created to help developers build and grow.
 
-### Clone the Template
+## Installation
 
-Start by cloning this repository:
+### Create a Slack App
 
-```zsh
-# Clone this project onto your machine
-$ slack create my-app -t slack-samples/bolt-python-custom-function-template
+1. Open [https://api.slack.com/apps/new](https://api.slack.com/apps/new) and
+   choose "From an app manifest"
+2. Choose the workspace you want to install the application to
+3. Copy the contents of [manifest.json](./manifest.json) into the text box that
+   says `*Paste your manifest code here*` (within the JSON tab) and click _Next_
+4. Review the configuration and click _Create_
+5. Click _Install_ button and _Allow_ on the screen that follows. You'll then be
+   redirected to the App Settings dashboard.
 
-# Change into the project directory
-$ cd my-app
+### Environment Variables
 
-# Install dependencies
-$ pip install -r requirements.txt
-```
+Before you can run the app, you'll need to store some environment variables.
 
-## Running Your Project Locally
+1. Open your apps configuration page from this list, click **OAuth &
```

Comment:
```suggestion
1. Open your apps setting page from this list, click **OAuth &
```

whoop found another!

Reply:
Good catch 💯 I should probably update this in all our other samples 🤔